pkg/webui/console/components/rights-group/index.js

```diff
@@ -75,16 +77,17 @@ const computeProps = props => {
     value.includes(RIGHT_ALL) ||
     derivedPseudoRight.some(derivedRight => value.includes(derivedRight))
 
-  // Determine the current grant type.
-  const grantType = hasPseudoRightGranted ? 'pseudo' : 'individual'
+  // Determine whether read-only rights are granted.
+  const readOnlyRights = derivedRights.filter(right => right.includes('_READ'))
+  const hasReadOnlyRights = isEqual(readOnlyRights.sort(), value.sort())
```

Comment:
You dont really need to filter first, then sort each list and compare every element. You can replace this by something like this:
```suggestion
  const hasReadOnlyRights = !hasPseudoRightGranted && value.every(right => right.includes('_READ'))
```